services-directions/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -272,13 +273,100 @@ public static Builder builder() {
    * each separated by  ; . Values can be any string and total number of all characters cannot
    * exceed 500. If provided, the list of waypoint_names must be the same length as the list of
    * coordinates, but you can skip a coordinate and show its position with the  ; separator.
-   * @return  a string representing names for each waypoint
+   *
+   * @return a string representing names for each waypoint
    * @since 3.3.0
    */
   @SerializedName("waypoint_names")
   @Nullable
   public abstract String waypointNames();
 
+  /**
+   * The type of bicycle, either `Road`, `Hybrid`, `City`, `Cross`, `Mountain`. The default type is
+   * Hybrid.
+   *
+   * @return the type of bicycle
+   * @since 4.1.0
+   */
+  @SerializedName("bicycle_type")
+  @Nullable
```

Comment:
We should add `@DirectionsCriteria.BicycleType` as in https://github.com/mapbox/mapbox-java/blob/a0c9ae45af2fd8b89818aa9660cf447773c944a5/services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java#L333